guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,233 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 In-Place Using Leapp
+
+Use this procedure to upgrade your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9.
```

Comment:
Let's try to make the introduction a bit more descriptive while also removing self-referential language ("this procedure").

```suggestion
When upgrading your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9, use the Leapp tool to help detect and resolve issues that could prevent you from upgrading successfully.
```

Feel free to modify or expand my suggestion. The main point is to avoid duplicating the procedure heading and instead give users an idea of what we will be asking from them in the procedure.